db/migrations/0004_alter_movie_actor_alter_movie_genre.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.0.2 on 2024-11-26 21:49
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0003_cinemahall_moviesession'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='movie',
+            name='actor',
+            field=models.ManyToManyField(related_name='actor', to='db.Actor'),
+        ),
+        migrations.AlterField(
+            model_name='movie',
+            name='genre',
+            field=models.ManyToManyField(related_name='genre', to='db.Genre'),
```

Comment:
The `related_name` for the `genre` field should be `movies` instead of `genre` to correctly represent the reverse relation from `Genre` to `Movie`.